sources/dogtag/bin/reverse.rs

```diff
@@ -0,0 +1,43 @@
+/*!
```

Comment:
I don't think we need the full `dogtag` docs on each resolver, maybe just some info on the resolver itself?